src/coreclr/gc/gcinterface.h

```diff
@@ -722,7 +722,7 @@ class IGCHeap {
 
     // Returns the generation in which obj is found. Also used by the VM
     // in some places, in particular syncblk code.
-    virtual unsigned WhichGeneration(Object* obj) PURE_VIRTUAL
+    virtual unsigned WhichGeneration(Object* obj, bool maxGenForNonGcObjects = true) PURE_VIRTUAL
```

Comment:
Changing this method signature would be GC interface breaking change. It is something that have been trying to avoid.

Reply:
I just wanted to make sure tests are passed 🙂